source/guide_matomo.rst

```diff
@@ -0,0 +1,107 @@
+.. highlight:: console
+.. author:: GodMod <godmod@eqdkp-plus.eu>
+
+.. sidebar:: Logo
+
+  .. image:: _static/images/matomo.png
+      :align: center
+
+#########
+Matomo
+#########
+
+Matomo_ (former known as Piwik) is an open source website tracking tool (like Google Analytics) written in PHP and distributed under the GNU General Public License v3.0 licence.
```

Comment:
Maybe describe in one or two sentences why it makes sense to do the tracking by oneself and not by a third party. GDPR comes to mind.

Reply:
Added.